lib/LaTeXML/Core/Whatsit.pm

```diff
@@ -98,26 +98,32 @@ sub unlist {
 sub revert {
   my ($self) = @_;
   # WARNING: Forbidden knowledge?
-  # (1) provide a means to get the RAW, internal markup that can (hopefully) be RE-digested
-  #     this is needed for getting the numerator of \over into textstyle!
   # (2) caching the reversion (which is a big performance boost)
-  if (my $saved = !$LaTeXML::REVERT_RAW
-    && ($LaTeXML::DUAL_BRANCH
+  if (my $saved = ($LaTeXML::DUAL_BRANCH
```

Comment:
There would be *lots* of pass-thru code. And would be mostly irrelevant since it's only a Math thing (of course, math is mostly the place where we care about reversion; but also graphics).  Thought about it before, but it still seems best to keep it dynamic/secret.